packages/build/src/signable-compiler.ts

```diff
@@ -38,7 +47,10 @@ class SignableCompiler {
         // Custom env vars for sccache:
         AWS_ACCESS_KEY_ID: process.env.DEVTOOLS_CI_AWS_KEY,
         AWS_SECRET_ACCESS_KEY: process.env.DEVTOOLS_CI_AWS_SECRET
-      }
+      },
+      addons: [
```

Comment:
wow! this is all we need to add native modules? 😮

Reply:
Yes – the trickier bit here is modifying the kerberos build config so that it supports being used as a static library in the first place :)